TGC/common/countries.txt

```diff
@@ -705,8 +705,8 @@
 	AOT		= "countries/Aotearoa.txt"
 	AST		= "countries/Australia.txt"
 	FAS		= "countries/French Australia.txt"
-	NZL		= "countries/New Zealand.txt"
 	NZF		= "countries/French New Zealand.txt"
+	NZL		= "countries/New Zealand.txt"
```

Comment:
Reordering tags like this is generally not a good idea, especially when not adding/removing any tag, since the game will not try to reload the flagcache if the amount of tags is still identical and therefore many users will be stuck with a corrupt cache where the wrong flag shows for all nations.

Reply:
@Perfo-6 Has this been fixed already?